databricks_dbapi/sqlalchemy_databricks.py

```diff
@@ -0,0 +1,29 @@
+from databricks_dbapi import databricks
+from pyhive.sqlalchemy_hive import HiveDialect
+
+
+class DatabricksDialect(HiveDialect):
```

Comment:
Do we need to overwrite any HiveDialect class variables here? Should we set name="databricks" at least?

Reply:
I was on the fence about overriding the `name` and `driver` attributes. In the sql alchemy Dialect interface they define name and driver as:
```
    name
      identifying name for the dialect from a DBAPI-neutral point of view
      (i.e. 'sqlite')

    driver
      identifying name for the dialect's DBAPI
```
At the end of the day, the Connection class that is returned is from pyhive and the driver is still thrift so I opted to not change them. If the `connect` function for the databricks dbapi returned a custom Connection class, I think there would be a stronger case to change it. Thoughts?